File: CSharp/Typing/Division.cs
using static System.Console;
using static System.Math;

public class Program {
	public static void Main() {
		var contador = 63;
		decimal arrendondamento = contador / 50M;
		WriteLine(arrendondamento);
		WriteLine(Ceiling(arrendondamento));
	}
}

//https://pt.stackoverflow.com/q/380046/101

File: Java/Typing/Division.java
public class Main {
	public static void main (String[] args) {
		System.out.println(calculoPrevisao(new double[] { 10, 5.5, 7}));
	}
    public static double calculoPrevisao(double[] valores) {
        double[] values = new double[valores.length];
        return 2.0 / (values.length + 1);
    }
}

//https://pt.stackoverflow.com/q/252461/101
